content/en/agent/faq/kubernetes-state-cluster-check.md

```diff
@@ -0,0 +1,102 @@
+---
+title: How do I run the kubernetes_state check as a cluster check?
+kind: faq
+further_reading:
+- link: "agent/kubernetes"
+  tag: "Documentation"
+  text: "Kubernetes with Datadog"
+- link: "agent/autodiscovery"
+  tag: "Documentation"
+  text: "Using Autodiscovery with Kubernetes and Docker"
+- link: "agent/autodiscovery/clusterchecks"
+  tag: "Documentation"
+  text: "Running Cluster Checks with Autodiscovery"
+---
+
+The `kubernetes_state` check, which monitors `kube-state-metrics`, is enabled automatically by [Autodiscovery][1]. In larger Kubernetes clusters, this check may consume too much RAM. In this case, consider running the `kubernetes_state` check as a [cluster check][2].
+
+## Configuration
+
+1. Deploy `kube-state-metrics` with cluster check annotations on the service:
+
+	```
+	ad.datadoghq.com/service.check_names: '["kubernetes_state"]'
+	ad.datadoghq.com/service.init_configs: '[{}]'
+	ad.datadoghq.com/service.instances: |
+	    [
+	    {
+	        "kube_state_url": "http://%%host%%:8080/metrics",
+	        "prometheus_timeout": 30,
+	        "min_collection_interval": 30,
+	        "send_pod_phase_service_checks": false,
+	        "telemetry": true
+	    }
+	    ]
+	```
+
+2. Deploy the Datadog Cluster Agent with `DD_CLUSTER_CHECKS_ENABLED` set to `true`.
+
+3. Mount an empty file to override `/etc/datadog-agent/conf.d/kubernetes_state.d/auto_conf.yaml` in the Agent DaemonSet. Otherwise, the Autodiscovered check still runs and may still consume a significant amount of RAM. Configure Agent deployment to mount an empty directory in `/etc/datadog-agent/conf.d/kubernetes_state.d`
+
+	```
+	...
+	    volumeMounts:
+	    ...
+	      - name: empty-dir
+	      mountPath: /etc/datadog-agent/conf.d/kubernetes_state.d
+	    ...
+	  volumes:
+	    - name: empty-dir
+	    emptyDir: {}
+	    ...
+	```
+
+4. Deploy the Agent.
+
+Refer to the [Running Cluster Checks with Autodiscovery][2] documentation for more information. See options that begin with `clusterchecksDeployment` in the Helm chart [README.md][3].
```

Comment:
Maybe we could be more verbose about the options to use in the agent ?

Reply:
Yeah we should aim to explain all options, but i think for an FAQ this is ok. We'll see what follow up questions users ask, and go from there.